go.mod

```diff
@@ -0,0 +1,3 @@
+module magma
+
+go 1.16
```

Comment:
what is this file

Note that we can;t switch to go 1.16

Reply:
The file automatically generated by ide was submitted by mistake and has been deleted,thank you .